Tests/ELFParser/AWSLCAESV8.lean

```diff
@@ -0,0 +1,360 @@
+/-
+Copyright (c) 2024 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Author(s): Yan Peng
+-/
+import Tests.ELFParser.AWSLCCrypto
+
+-- Importing just the aesv8-armx.S.o file to avoid ADRP issue.
+-- Details: PC relative addressing are used for locating constants
+-- The address changes every time linking happens.
+-- We use the .o files to avoid having to deal with address change.
```

Comment:
```suggestion
-- Importing just the aesv8-armx.S.o file to avoid relying on the 
-- linker-generated addresses.
-- Details: PC-relative addressing is often used for locating constants.
-- The addresses can change every time linking happens, which
-- means that the 32-bit hex for instructions that use PC-relative offsets
-- can change. That, in turn, means that the program obtained by 
--`getSymbolInsts` can vary, which in turn causes `#guard_msgs` to fail.
--
-- We use the .o files to avoid having to deal with the repercussions of
-- such potential address changes.
```

Maybe elaborate a bit here, something along the lines of this suggestion (which admittedly is not great, but it's also a tad late for me :)

Reply:
Done. Adjusted the wording a bit.